### ISSUE TITLE
UserWarnung after execution of "Execution Task: Verification"

### ISSUE THREAD

**Issue by tillerKleinmann** [zincware/ZnPINNLab#1]
[/home/tkleiner/Projects/uni/icp/pinn/day2/ZnPINNLab/day2/tools.py:198](https://file+.vscode-resource.vscode-cdn.net/home/tkleiner/Projects/uni/icp/pinn/day2/ZnPINNLab/day2/tools.py:198): UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  _X = torch.tensor(torch.dstack((X,T))).float().to(device)
